third_party/xla_client/pjrt_computation_client.cc

```diff
@@ -585,6 +582,29 @@ PjRtComputationClient::ExecuteReplicated(
     data_handles.push_back(datas);
   }
 
+  auto mwait = std::make_shared<util::MultiWait>(1);
```

Comment:
I'd really like to consolidate these code paths somehow. I don't think it's doable within this PR though.

Reply:
the `MultiWait` is on upstream now, as long as computationClient can import pytorch code, we can kill our own version.